dangerfile.js

```diff
@@ -126,11 +115,13 @@ function git(args) {
 
   const percentToWarrentShowing = 1;
   const packagesToShow = results
+    .filter(r => r.prevFileSize && r.prevGzipSize)
```

Comment:
Can you explain why this is necessary?

It seems like if I add a package, I'd like to see its size reported.

And for this PR, I wouldn't expect a "new package" to get reported because this PR did not add a new package.

Reply:
Sure, I can remove it again. 

Currently, it compares the new package against a non existent one in the previous results json and says how it is infinitely bigger (that was the message from last night) - which is true strictly speaking. I've added some protection for that 👍